ops/terraform/modules/stateful/main.tf

```diff
@@ -423,3 +423,51 @@ module "etl" {
   kms_key_id          = data.aws_kms_key.master_key.arn
   log_bucket          = module.admin.id
 }
+
+# IAM policy to allow read-write access to ETL bucket
+#
+resource "aws_iam_policy" "etl_rw_s3" {
+  name        = "bfd-${local.env_config.env}-etl-rw-s3"
+  description = "ETL read-write S3 policy"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "ETLRWKMS",
+      "Action": ["kms:Decrypt"],
+      "Effect": "Allow",
+      "Resource": ["${data.aws_kms_key.master_key.arn}"]
+    },
+    {
+      "Sid": "ETLRWBucketList",
+      "Action": ["s3:ListBucket"],
+      "Effect": "Allow",
+      "Resource": ["${module.etl.arn}"]
+    },
+    {
+      "Sid": "ETLRWBucketActions",
+      "Action": [
+        "s3:GetObject",
+        "s3:PutObject",
+        "s3:DeleteObject"
```

Comment:
Can you remove this one? Not sure they need it, and leaving it off might prevent oopses.